### PR TITLE
Main page: Structure

### PR DESCRIPTION
**Pull request by manuelsanchez2**
In this pull request I tried in two different ways to achieve the horizontal carousel. 

After having used the Owl Carousel, I decided to use simple HTML and CSS instead of that library since the library was quite old (2018) and do not receive updates and support. 

Screenshot 2020-09-14 at 17 19 30
---
Screenshot 2020-09-14 at 17 19 44

Next objectives here:
-Learn how to link the share button with a share function on React that enables sharing the App on different platforms.
- Improve the general code.

### CODE REVIEW

File: client/package.json [hunk 1]
```diff
@@ -16,13 +16,18 @@
     "@testing-library/react": "^9.5.0",
     "@testing-library/user-event": "^7.2.1",
     "babel-loader": "^8.1.0",
+    "dotenv": "^8.2.0",
```
Comment (lmachens) [inline client/package.json]: I don't think you need to install `dotenv` on the client. It is already bundled in your React app. See https://create-react-app.dev/docs/adding-custom-environment-variables

File: client/package.json [hunk 2]
```diff
@@ -16,13 +16,18 @@
...
     "prop-types": "^15.7.2",
     "react": "^16.13.1",
     "react-dom": "^16.13.1",
     "react-is": "^16.13.1",
+    "react-mailchimp-form": "^1.0.2",
+    "react-mailchimp-subscribe": "^2.1.0",
+    "react-owl-carousel": "^2.3.1",
```
Comment (lmachens) [inline client/package.json]: Can be removed

File: client/package.json [hunk 3]
```diff
@@ -16,13 +16,18 @@
...
     "react-query": "^2.15.4",
     "react-router-dom": "^5.2.0",
     "react-scripts": "3.4.3",
+    "react-share": "^4.2.1",
```
Comment (lmachens) [inline client/package.json:27]: Can be removed

File: client/package.json [hunk 4]
```diff
@@ -16,13 +16,18 @@
     "@testing-library/react": "^9.5.0",
     "@testing-library/user-event": "^7.2.1",
     "babel-loader": "^8.1.0",
+    "dotenv": "^8.2.0",
     "prop-types": "^15.7.2",
     "react": "^16.13.1",
     "react-dom": "^16.13.1",
     "react-is": "^16.13.1",
+    "react-mailchimp-form": "^1.0.2",
+    "react-mailchimp-subscribe": "^2.1.0",
```
Comment (lmachens) [inline client/package.json]: Not sure, but I think this can be removed as  well :)

File: client/public/index.html [hunk 5]
```diff
@@ -27,6 +27,7 @@
     <title>React App</title>
   </head>
   <body>
+    <script src="https://code.jquery.com/jquery-3.2.1.slim.min.js"></script>
```
Comment (lmachens) [inline client/public/index.html]: I guess this is related to the `react-owl-carousel` package and should be removed

File: client/src/GlobalStyles.js [hunk 6]
```diff
@@ -109,6 +109,103 @@ const GlobalStyles = () => {
...
+        /* MAIN PAGE  */
+
+        .main--main {
+          flex: 1;
+          padding: 0 1em 0 1em;
+        }
+
+        /* MAIN SUBSCRIBE SECTION  */
+
+        .subscribe__section {
```
Comment (lmachens) [inline client/src/GlobalStyles.js:121]: You define a lot of CSS classes in addition to your styled components. I wouldn't mix it and focus on styled components or CSS, not both.

File: client/src/components/Header/Hamburger.js [hunk 7]
```diff
@@ -4,27 +4,26 @@ import styled from "@emotion/styled";
...
 `;
 
 const StyledBurger = styled.div`
   width: 2rem;
   height: 2rem;
-
-  position: absolute;
-  z-index: 999;
-  display: none;
+  /* position: fixed; */
```
Comment (lmachens) [inline client/src/components/Header/Hamburger.js]:
```suggestion
```

File: client/src/pages/Home/Home.js [hunk 8]
```diff
@@ -1,32 +1,182 @@
...
+
+import Mailchimp from "react-mailchimp-form";
+
 import Header from "../../components/Header";
 import Footer from "../../components/Footer";
+import styled from "@emotion/styled";
+import PictureTest from "../../assets/images/picture-test-1.jpg";
+import PictureTest2 from "../../assets/images/picture-test-2.jpg";
+import PictureTest3 from "../../assets/images/picture-test-3.jpg";
+import instagramSrc from "../../assets/icons/social-instagram.svg";
```
Comment (lmachens) [inline client/src/pages/Home/Home.js]: You are using different naming conventions

**Review by lmachens — CHANGES_REQUESTED** [review]
The deployed pages are looking good! Good job :).

File: client/package.json (hunk 2 above)
Comment (manuelsanchez2) [reply client/package.json]: The mailchimp subscribe is being used in the form, it is already connected with the Doiche Living MailChimp account. When it comes to the other two, yeah! I am going to delete them! thanks!

File: client/package.json (hunk 1 above)
Comment (manuelsanchez2) [reply client/package.json]: Thanks!

File: client/package.json (hunk 3 above)
Comment (manuelsanchez2) [reply client/package.json:27]: I was trying to create a link in the share button that opens a new tab asking the use for sharing the app on different platforms. That's why I installed react-share, but I think it is not the proper package for that issue. I will delete it!

File: client/public/index.html (hunk 5 above)
Comment (manuelsanchez2) [reply client/public/index.html]: Thanks! Your are right!

File: client/src/GlobalStyles.js (hunk 6 above)
Comment (manuelsanchez2) [reply client/src/GlobalStyles.js:121]: Sure! I will move almost everything to styled components! Thanks for the advice!

File: client/src/pages/Home/Home.js (hunk 8 above)
Comment (manuelsanchez2) [reply client/src/pages/Home/Home.js]: True!

**Pull request merged**